### PR TITLE
Update README

### LINKED ISSUE #20: update README
- `README.md` is not up to date
- `README.md` should be up to date with corresponding code
- `README.md` should be updated

aka Make docs useful for other people.

### PR DESCRIPTION
**Pull request by svenwltr**
Closes #20

### CODE REVIEW

**Comment by svenwltr** [comment]
@rebuy-de/prp-consul-java-client

**Comment by ghost** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -3,32 +3,133 @@ consul-java-client
 
 Consul client for our Java services.
 
+* minimized bootstrapping amount
+* easy Spring Boot and Spring support
+* only one health check is supported
```
Comment (florian-zeidler) [inline README.md]: Great feature?

**Comment by florian-zeidler** [comment]
:+1:  Documentation is good.

File: README.md (hunk 1 above)
Comment (svenwltr) [inline README.md]: Change it. Is this better?

Comment (florian-zeidler) [inline README.md]: I would have written something like, because of the underlying client ... Because it is indeed a limitation.

**Pull request merged**